.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,51 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello and thanks for contributing!
+
+        Before continuing, can you check if the answer is available within:
+
+        * Guides and examples - https://www.mapbox.com/ios-sdk/navigation/
+        * API reference - https://www.mapbox.com/mapbox-navigation-ios/navigation/
+        * The example app - https://github.com/mapbox/navigation-ios-examples
+  - type: dropdown
+    attributes:
+      label: "Mapbox Navigation SDK version"
+      options:
+        - 2.0.0-beta.19
```

Comment:
Does this mean that we'd need to update this template with every release? (Or at least every major release?)

Reply:
Suggestions are welcomed :) I would list here max 7 most common versions. Including beta here is temporary as we don't have release yet.

Reply:
A freeform textbox is probably OK for this field. Once v2.0.0 is released, most bug reports will be against a version that’s easier to type. 🙂